static-assets/scripts/admin.js

```diff
@@ -214,8 +214,8 @@
         });
       };
 
-      this.getSpecificAudit = function(auditId) {
-        return $http.get(audit(auditId));
+      this.getSpecificAudit = function(siteName) {
+        return $http.get(audit(siteName));
```

Comment:
The API changed to retrieve by name instead of ID?

Reply:
yes it did change, it does retrieve by name

Reply:
@sumerjabri was it intended to change the API to retrieve by name instead of id?

Reply:
You can show by name, but API keys off of ID.